contrib/babelfishpg_tsql/src/guc.c

```diff
@@ -1232,6 +1250,16 @@ define_custom_variables(void)
 							 PGC_SUSET,
 							 GUC_NO_SHOW_ALL | GUC_NOT_IN_SAMPLE | GUC_DISALLOW_IN_AUTO_FILE,
 							 NULL, NULL, NULL);
+
+	DefineCustomBoolVariable("babelfishpg_tsql.disable_storing_init_privs",
+							 gettext_noop("Disable storing initial privileges while "
+										  "CREATE/ALTER Babelfish TSQL extension"),
+							 NULL,
+							 &pltsql_disable_storing_init_privs,
+							 false,
+							 PGC_USERSET,
```

Comment:
why are we opening it to any user even with checks inside assign_disable_storing_init_privs? why not just SU?

Reply:
Non-superuser can CREATE/ALTER extension as well. that's why I kept it open to all the user.

Reply:
Confirmed TSQL extension can only be installed by SUPERUSERs as configured in it's control file. Changed it to only SU settable.